packages/@sanity/components/src/previews/MediaPreview.js

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable complexity */
 import PropTypes from 'prop-types'
 import React from 'react'
 import styles from 'part:@sanity/components/previews/media-style'
+import ProgressCircle from 'part:@sanity/components/progress/circle'
 import SvgPlaceholder from './common/SvgPlaceholder'
```

Comment:
`SvgPlaceholder` is no longer in use in this file, so this import should be removed.

Reply:
👍